dali/pipeline/executor/pipelined_executor.cc

```diff
@@ -47,12 +47,12 @@ void PipelinedExecutor::SetupStageOutputsForGraph() {
   // Make a set of the outputs names for quick lookup
   std::set<string> output_set(output_names_.begin(), output_names_.end());
 
-  for (int i = 0; i < graph_->NumSupportOp(); ++i) {
+  for (int i = 0; i < graph_->NumOp(DALIOpType::DALI_SUPPORT); ++i) {
```

Comment:
Apparently, ye olden way of goin' through the graph is still quite firmly rooted in our source base. Wouldn't you mind resurrecting the old functions as a convenient way of accessing the subsets of the graph related to particular `DALIOpType`?

Reply:
I am trying to get rid of ye olden way of processing graph in consecutive PRs. This was to get rid of writing the same code with distinct function names 4 times.